src/main/java/rs/hybridit/controller/BookController.java

```diff
@@ -0,0 +1,74 @@
+package rs.hybridit.controller;
+
+
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import rs.hybridit.dto.BookDto;
+import rs.hybridit.dto.LibraryDto;
+import rs.hybridit.model.Book;
+import rs.hybridit.model.Library;
+import rs.hybridit.service.BookService;
+
+@RestController
+@RequestMapping(value = "/books", produces = MediaType.APPLICATION_JSON_VALUE)
+public class BookController {
+
+	private BookService bookService;
+
+	public BookController(BookService bookService) {
+		this.bookService = bookService;
+	}
+
+	@PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
+	@ResponseStatus(HttpStatus.CREATED)
+	public Book create(@RequestBody BookDto bookDto) {
+		return bookService.create(new Book(bookDto));
+	}
+
+	@GetMapping(value = "/{id}")
+	@ResponseStatus(HttpStatus.OK)
+	public Book getBook(@PathVariable Long id) {
+		Book book = bookService.getOne(id);
```

Comment:
inline this, no need for a variable which is going to be returned in the next line